packages/core/types/src/notification/providers/local.ts

```diff
@@ -0,0 +1 @@
+export interface LocalNotificationServiceOptions {}
```

Comment:
This is just a placeholder, I'll add the interface of some of the providers we will support out of the box